content/intro-to-storybook/vue/pt/conclusion.md

```diff
@@ -3,31 +3,32 @@ title: 'Conclusão'
 description: 'Fusão de todo o conhecimento adquirido e aprendizagem de técnicas extra de Storybook'
 ---
 
-Parabéns! Acabaste de criar o teu primeiro interface de utilizador com Storybook. Durante esta jornada aprendeste como construir, compor, testar e implementar componentes de interface de utilizador.
-Se estiveste a seguir o tutorial, o repositório e o Storybook implementado irá ser idêntico a este:
+Parabéns! Você criou sua primeira interface do usuário no Storybook. Ao longo do caminho, você aprendeu a criar, compor, testar e implantar componentes da interface do usuário. Se você seguiu o tutorial, seu repositório e o Storybook implantado devem ficar assim:
```

Comment:
Probably a better wording for this could be:
"Parabéns! Acabou de criar o primeiro interface de utilizador com o Storybook. Ao longo do caminho, aprendeu a criar, compor, testar e implementar componentes de interface de utilizador. Se seguiu o tutorial o repositório e o Storybook deverão ficar assim"

Reasoning behind this:
- Avoid usage of "você" to maintain a professional tone to the document without breaking verb tenses and grammar involved. 
-  i know about the usage of "utilizador", in brazilian it's customary to use "usuário". "Utilizador" is a valid word that is recognized by both readers and once again if the end goal of the documentation is to reach the maximum number of people it would be best to use some words/terms that are more "universal" to avoid "tipping the scale to one plate"(pardon the bad pun).
- Removed both "implementado" and "implantado", to avoid any language specialization, when the reader reaches this point it knows that it has a fully functional code repository in GitHub and also its own Storybook deployed in netlify.

Reply:
show, value for the collaboration.